recipes/asimov/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "asimov" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 160df0c977cf5c65794f244957c7f4436d250bbf63fd944654219bf29b9114e3
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - locutus=asimov.locutus:cli
```

Comment:
Star trek on asimov? 
![image](https://user-images.githubusercontent.com/5234427/220176329-712aa174-ccb4-4b8d-bcd7-e82f930ed45b.png)

Reply:
It's a long story, but we'll eventually deprecate it.